wagtail_wordpress_import/importers/wordpress.py

```diff
@@ -125,6 +125,27 @@ def run(self, *args, **kwargs):
                             wp_post_id=wordpress_item.cleaned_data.get("wp_post_id")
                         )
                     except self.page_model_class.DoesNotExist:
+                        try:
+                            # We are creating a new page as the wp_post_id doesn't exist
+                            # however we may have a collision of slugs between page and post types
+                            # https://core.trac.wordpress.org/ticket/13459
+                            slug_post = self.page_model_class.objects.get(
```

Comment:
note if wp_post_parent support from #176 was added, this would need to consider a slug_post with the same parent as the page and not just any slug page